forumdisplay.php

```diff
@@ -1494,6 +1494,12 @@
 
 	$prefixselect = build_forum_prefix_select($fid, $tprefix);
 
+	// Populate Forumsort
+	if($threadcount > 0)
+	{
+		eval("\$forumsort = \"".$templates->get("forumdisplay_forumsort")."\";");
```

Comment:
I think that `$forumsort` need initialization with empty string, above this if. Otherwise there will be a notice for case that threadcount is 0.

Reply:
Hmm, I never stumbled into that problem, but taking precaution is a good idea. I'll make the changes.